src/sentry/api/endpoints/api_tokens.py

```diff
@@ -24,11 +24,28 @@
 ALLOWED_FIELDS = ["name", "tokenId"]
 
 
-class ApiTokenNameSerializer(serializers.Serializer):
+def _get_appropriate_user_id(request: Request) -> int:
```

Comment:
Any reason why we moved this up?

It makes this PR seem like there's a lot more changes 😅

Reply:
Oh i see.... to make it a global 🤔 

If it was a class method before - we could just call it on the class?

eg
```
ApiTokenSerializer._get_appropriate_user_id(request)
```
?

Is this not acceptable?

Reply:
Ah you're right, changed it to be a static method